docs/sources/auth/azuread.md

```diff
@@ -89,6 +89,13 @@ To enable the Azure AD OAuth2, register your application with Azure AD.
 
 1. Click on **Users and Groups** and add Users/Groups to the Grafana roles by using **Add User**.
 
+1. Under Azure AD / App Registrations / Grafana / **Token Configuration**, click **Add Optional Claim** and select Type ***ID** and Claim **email**.
+
+1. Under Azure AD / App Registrations / Grafana / **API Permissions**, Add following permissions:
+    * **Directory.AccessAsUser.All** from **Azure AD Directory Graph**
```

Comment:
Should it be `Delegated permissions` or `Application permissions` (guess, delegated)? Also, Azure AD Directory Graph is deprecated, and it's suggested to use Microsoft Graph APIs.

Reply:
![Screenshot from 2022-02-07 11-59-55](https://user-images.githubusercontent.com/4932851/152756840-76d739bb-42ff-478a-aa47-d996ce313a59.png)

![Screenshot from 2022-02-07 12-00-23](https://user-images.githubusercontent.com/4932851/152756894-d70ea660-acfb-4f3e-a6c6-6074b7ea0748.png)